src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -24,9 +24,27 @@
 
 /**
  * @author Andreas Möller <am@localheinz.com>
+ * @author SpacePossum
  */
 final class NativeFunctionInvocationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /**
+     * @internal
+     */
+    const SET_ALL = '@all';
+
+    /**
+     * Subset of SET_INTERNAL.
```

Comment:
What about adding a link to https://github.com/php/php-src/blob/master/Zend/zend_compile.c with the function name `zend_try_compile_special_func` to look at, or simply direct link https://github.com/php/php-src/blob/php-7.2.4/Zend/zend_compile.c#L3872? Someone more interested what are "these strange" `SET_COMPILER_OPTIMIZED` functions will probably check the fixer code first.

Reply:
:+1:  added